nni/retiarii/trial_entry.py

```diff
@@ -0,0 +1,15 @@
+"""
+Entrypoint for trials.
+
+Assuming execution engine is BaseExecutionEngine.
+"""
+import os
+
+from .execution.base import BaseExecutionEngine
+from .execution.cgo_engine import CGOExecutionEngine
+
+if __name__ == '__main__':
+    if os.environ.get('CGO') == 'true':
```

Comment:
Where is this set?

Reply:
Which set? If user sets "CGO=true" in the environment variables, it uses CGOExecutionEngine.

Reply:
There are training services other than "local", where trials will run on ssh server or cloud. So environment variables cannot be set by user.

Reply:
yes, this part does not work for other training services. we should refactor this part.
currently, retiarii only supports local training service...

Reply:
We can add "CGO=true" in the trial_command of RetiariiExeConfig if cross-graph optimization is enabled.